### ISSUE TITLE
CSS not injected

### ISSUE THREAD

**Issue by Kore-Development** [jiahaog/nativefier#794]
### Description

I tried run the command `nativefier "https://www.discordapp.com/activity" --inject /Users/[REMOVED]/nox.css ~/Desktop` on macOS Majave and the css doesn't even load (from what I know). The css works with [userstyles.org](http://userstyles.org) just not Nativefier. Is there something I'm doing wrong?

The CSS
```css
@import url('https://cdn.jsdelivr.net/gh/rauenzi/Nox@master/release/import.css');

[class^="downloadAppButton"],[class^="guildSeparator"] {
    display: none !important;
}
```

### Steps to reproduce issue
Run the command and have the css file.

### Details

- Are you nativefying a *public* website? Yes
- Feature request? Have you looked at `nativefier --help` to see if an existing option could fit your needs? I looked.
- Full nativefier command used to build your app: `nativefier "https://www.discordapp.com/activity" --inject /Users/[REMOVED]/nox.css ~/Desktop`
- Version of Nativefier (run `nativefier --version`): `7.6.12`
- Version of node.js (run `node --version`): `10.12.0`
- OS: `macOS Mojave`
- Error message / stack trace (if any): None

**Comment by dwillie** [comment]
I've also experienced this when using a userstyles CSS file with `--inject`.

As a stop-gap I've found that using the userscript version of a userstyles.org theme (JS, not CSS) has worked just fine. There's a link on the userstyles page for a style underneath the patch notes.

**Comment by Kore-Development** [comment]
> I've also experienced this when using a userstyles CSS file with `--inject`.
> 
> As a stop-gap I've found that using the userscript version of a userstyles.org theme (JS, not CSS) has worked just fine. There's a link on the userstyles page for a style underneath the patch notes ("Install style as a userscript", which just opens generated JS as raw text).

It's not a style from userstyles.org I just meant to say that I used it to load it.

**Comment by ravenyu** [comment]
Having the same issue on macOS Catalina as well.

**Comment by kocoai** [comment]
please release the PR https://github.com/jiahaog/nativefier/pull/837 that fixed this bug

**Comment by pablochupin** [comment]
There is still same issue. macOS Catalina. nativefier 7.7.0.

**Comment by icyavocado** [comment]
I ran into this issue, solved by using this js script to import css

`var link = document.createElement('link');

// set the attributes for link element
link.rel = 'stylesheet';

link.type = 'text/css';

link.href = 'https://cdn.jsdelivr.net/gh/0x0049/Rocket.Chat.Dark@master/dist/dark.css';

// Get HTML head element to append
// link element to it
document.getElementsByTagName('HEAD')[0].appendChild(link);`

**Comment by ronjouch** [comment]
Should be fixed with 7.7.1 recently released.
